source/ScreensaverHero.hx

```diff
@@ -0,0 +1,27 @@
+package;
+
+import flixel.FlxG;
+import flixel.util.FlxColor;
+import flixel.FlxSprite;
+import Math;
+
+class ScreensaverHero extends FlxSprite 
+{
+    private var MovementDirectionX = FlxG.random.float(-2, 2);
+    private var MovementDirectionY = FlxG.random.float(-2, 2);
+    
+    public function new()
+    {
+        super();
+        makeGraphic(50, 80, FlxColor.WHITE, false);
```

Comment:
Another spot with magic numbers, let's get them replaced with some kind of variable (preferably something static and unchangeable)